### ISSUE TITLE
Change adapting text to "blocks of text"

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wcag21#576]
We've had 2 reputable commenters who are veteran full time accessibility professionals and testers, ask that this SC get changed to "Blocks of Text", [Jan Richards of the IDRC](https://github.com/w3c/wcag21/issues/551), and [Aiden from TD Bank](https://github.com/w3c/wcag21/issues/469). 

I'd like to add my voice to that concern. I was the author of SC 1.4.8 in WCAG 2.0. And the "blocks of text" language was carefully negotiated, I never could have got through without that. I think we should carefully consider "Blocks of Text". Without that, there are a lot of variables, and possible confusion. It might become a stumbling block for the SC in CR.

**Comment by lauracarlson** [comment]
Hi @DavidMacDonald ,

@alastc , @goodwitch , Alastair's colleague, and I tested [67 pages between July 25 and August 3, 2017](https://www.w3.org/WAI/GL/wiki/Results_of_Bookmarklet_Tests_for_Issue_78) that included top 50 Alexa pages and some smaller pages, the combined metrics specified in the Adapting Text SC should be feasible in WCAG 2.1 for HTML documents. Based on that evaluation spacing was able to adapt without loss of content or functionality across a wide range of sites. 

What types of content outside of blocks of text are you worried about? Mega menus? I just checked the mega menus on the following sites with the bookmarklet and they all seem to pass:

* [Columbia](https://www.columbia.com/)
* [Callaway Golf](http://cmp.callawaygolf.com/)
* [Puma](http://us.puma.com/)
* [The Sak](http://www.thesak.com/)
* [Adidas](http://www.adidas.com/us)
* [Northerntool](https://www.northerntool.com/)
* [Walmart](https://www.walmart.com/)
* [Starbucks](https://www.starbucks.com/)
* [Adobe's Accessible Mega Menu](https://adobe-accessibility.github.io/Accessible-Mega-Menu/)

Thanks,
Laura

**Comment by DavidMacDonald** [comment]
@lauracarlson 
There are hundreds of types of content that are not blocks of text. 
I'm concerned about left navs, right navs, mega menus, popup widgets, date pickers, etc... I can live without going to "blocks of text" but do want to draw attention to two reputable commenters, and it has also been a concern of mine from the beginning.

**Comment by alastc** [comment]
Hi David, 

I was also concerned, until I saw how the testing went. We have lots of those types of content in the test pages.  We rarely got issues in those types of components.

The commenters are reputable, but we have tested many sites and have data. The most common issues were around small 'blocks of text' (within the current definition), so I don't agree with changing the scope now.

If someone can find examples (or even one?) where it could not be met then we should reconsider, but otherwise I can't see why we should change it?

It aligns with good practice as well, as having no buffer around text (like the 'logo' on my site ;-) is not a good idea, as fonts can take different amounts of space on different systems.

**Comment by lauracarlson** [comment]
Hi @DavidMacDonald, and @alastc ,

[David wrote](https://github.com/w3c/wcag21/issues/576#issuecomment-345868180):

> There are hundreds of types of content that are not blocks of text.
> I'm concerned about left navs, right navs, mega menus, popup widgets, date pickers, etc... I can live without going to "blocks of text" but do want to draw attention to two reputable commenters, and it has also been a concern of mine from the beginning.

Thank you, David. If you have specific examples that don't currently pass that would be would be very helpful.

I am sure you are aware that WCAG 2.1 introduces new requirements, some of those will not be met by WCAG 2.0 compliant sites. This is to be expected. They will still be WCAG 2.0 compliant and can then make efforts to become WCAG 2.1 compliant as well.

But with that said, do you think it would be a good idea to add a note something such as:

Editors Note: The Working Group seeks examples in the wild of text content outside of blocks of text where it would be impossible to meet this SC. The SC's scope may be narrowed based on implementation testing.

**Comment by DavidMacDonald** [comment]
I think the note makes good sense.

**Comment by awkawk** [comment]
Thank you for your comment.

Based on an evaluation of 67 pages tested between July 25 and August 3, 2017 that included top 50 Alexa pages and some smaller pages, the combined metrics specified in the Adapting Text SC should be feasible in WCAG 2.1 for HTML documents. Spacing is able to adapt without loss of content or functionality across a wide range of sites. In addition recently 9 mega menus were tested and passed the SC.

The Working Group will be going through the implementation process and demonstrating the implementability of all the new SCs. If you know of examples in the wild of text in UI - (e.g. menus, etc) where it would not be possible to meet the SC, please add them to this issue.

We are tagging this issue with the "Implementation Follow-up" label to ensure that it is re-reviewed at that point in our process. In addition we will add the following editor's note:

"Editors Note: The Working Group seeks examples in the wild of text content outside of blocks of text where it would be impossible to meet this SC. The SC's scope may be narrowed based upon implementation testing."
